features/ai-and-embeddings.mdx

```diff
@@ -115,22 +122,30 @@ SELECT title,
        vector_extract(embedding),
        vector_distance_cos(embedding, vector32('[0.064, 0.777, 0.661, 0.687]'))
 FROM movies
-ORDER BY 
+ORDER BY
        vector_distance_cos(embedding, vector32('[0.064, 0.777, 0.661, 0.687]'))
 ASC;
 ```
 
-<Note>
-  The `vector_distance_cos` function calculates the cosine **distance**, which
-  equals to 1 - [cosine
-  similarity](https://en.wikipedia.org/wiki/Cosine_similarity). Therefore, a
-  smaller distance indicates that the vectors are closer to each other.
-</Note>
-
 </Step>
 
 </Steps>
 
+### Understanding Distance Results
+
+The `vector_distance_cos` function calculates the cosine distance, which equals 1 - [cosine
+similarity](https://en.wikipedia.org/wiki/Cosine_similarity). Therefore:
+
+- A distance close to 0 (including very small negative numbers like `-6.394e-14`) indicates that the vectors are nearly identical or exactly matching
+- Values closer to 1 indicate that the vectors are dissimilar
```

Comment:
cosine distance actually lies in range `[0..2]` - so values closer to 2 indicate that vectors are opposite, and values close to 1 indicate that vectors are orthogonal